ttnn/cpp/ttnn/operations/conv/conv_transpose2d/conv_transpose2d.hpp

```diff
@@ -0,0 +1,60 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+#include "ttnn/operations/conv/conv2d/conv2d.hpp"
+
+namespace ttnn {
+
+namespace operations::conv {
+namespace conv_transpose2d {
+
+
+struct ConvTranpose2dOperation{
+    static std::tuple<ttnn::Tensor, uint32_t, uint32_t, ttnn::Tensor, std::optional<ttnn::Tensor>> invoke(
```

Comment:
Please do this
```
using MeaningOfThisUint32 = uint32_t;
using MeaningOfThatUint32 = uint32_t;
using Result = std::tuple<ttnn::Tensor, MeaningOfThisUint32, MeaningOfThatUint32, ttnn::Tensor, std::optional<ttnn::Tensor>>;
```

Lets reduce some duplication and rely on this using across the code to reduce pain..